src/main/java/org/openrewrite/java/spring/framework/MigrateHandlerInterceptor.java

```diff
@@ -30,6 +29,9 @@
 
 public class MigrateHandlerInterceptor extends Recipe {
 
+    public static final String HANDLER_INTERCEPTOR_ADAPTER = "org.springframework.web.servlet.handler.HandlerInterceptorAdapter";
+    public static final String HANDLER_INTERCEPTOR = "org.springframework.web.servlet.HandlerInterceptor";
```

Comment:
There's no need for these to be public right?
```suggestion
    private static final String HANDLER_INTERCEPTOR_ADAPTER = "org.springframework.web.servlet.handler.HandlerInterceptorAdapter";
    private static final String HANDLER_INTERCEPTOR = "org.springframework.web.servlet.HandlerInterceptor";
```